src/contents/en-gb/parsec.yaml

```diff
@@ -0,0 +1,66 @@
+pageIntro:
+  title: What is PARSEC?
+  desc: PARSEC is a Protocol for Asynchronous, Reliable, Secure and Efficient Consensus and is the world's first open source ABFT protocol that can be used in a truly permissionless setting created by MaidSafe.
+networkUse:
+  title: Why does the Network need it?
+  para1: As a global decentralised data and communications network, the SAFE Network has to be public and permissionless.
+  para2: Unlike centralised systems there is no central body to act as a final authority on the state of the Network.  Nodes is a decentralised system such as the SAFE Network must therefore agree on Network actions and state amongst themselves. Other decentralised systems such as BitCoin use blockchain to achieve consensus.  However, there is a number of drawbacks with a blockchain based system notably the speed and cost as blockchain grows - this is not a scalable option.
+bigDeal:
+  title: Why is it a big deal?
+  para1: As far as we know no other project has managed to ensure ABFT consensus in permissionless environments.  This is a technology that will power the SAFE Network, the only truly next-generation decentralised autonomous internet that promises privacy, security and freedom by default.

```

Comment:
environments. This is ~a~ technology that will power the SAFE Network ~,~ `:`

Reply:
done